### DISCUSSION TITLE
How do Train Custom Languages rather than English!

### DISCUSSION THREAD

**Discussion by robadugna2** [JarodMica/StyleTTS-WebUI#9] [Ideas]
i want to train a custom low resource language which is having a unique grapheme/chars...  can i please? if i can, how do i do by using my own g2p & tokenizer in this way! 

Please Guide me as much as u can!

Thank you very much!

**Comment by JarodMica** [comment]
This is something that is possible, but it's not too straightforward. Might be a feature implemented in the future, but you can look at the official styletts repo for this as a lot of people have talked about this there.
